Jenkinsfile

```diff
@@ -39,12 +39,7 @@ pipeline{
 								installAzureCli()
 							}
 						}
-						stage ('Download assets') {
-							steps {
-								sh "mkdir ${env.HOME}/workloads"
-								sh 'az storage blob download --container-name private-images --file "$HOME/workloads/OVMF-83041af43c.fd" --name OVMF-83041af43c.fd --connection-string "$AZURE_CONNECTION_STRING"'
-							}
-						}
+
```

Comment:
Well you can also remove the `Install azure-cli` stage from both `Worker build` and `Worker Build (musl)`